### ISSUE TITLE
Contrast within admin-side selects

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1152]
The arrows on admin-side selects have very low contrast (1.6:1, should be 3:1):

<img width="392" height="147" alt="Select that reads 'Quick Filter', followed by a downward arrow" src="https://github.com/user-attachments/assets/b5c24a90-e331-410c-bccf-910a3dcf9649" />
